### PR TITLE
[REFACTORING] ID tag expand abbreviation to full name

### PR DESCRIPTION
**Pull request by battermann**
## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: libs/types-common/src/Data/Id.hs [hunk 1]
```diff
@@ -87,54 +87,62 @@ import Servant (FromHttpApiData (..), ToHttpApiData (..))
...
-idTagName A = "Asset"
-idTagName C = "Conv"
-idTagName I = "Invitation"
-idTagName U = "User"
-idTagName P = "Provider"
-idTagName S = "Service"
-idTagName T = "Team"
-idTagName STo = "ScimToken"
+idTagName Asset = "Asset"
+idTagName Conversation = "Conv"
```
Comment (elland) [inline libs/types-common/src/Data/Id.hs:102]: Kind of annoying having this to basically replace a `Show` instance, now after the refactor, but don't want to pile up more stuff for you to do 👍

File: libs/types-common/src/Data/Id.hs [hunk 2]
```diff
@@ -179,7 +187,7 @@ instance Read (Id a) where
 instance FromByteString (Id a) where
   parser = do
     match <-
-      -- we only want the matching part of the ByteString, so the parser doesn't
+      -- we only want the matching part of the ByteString, so the parser doesn'Team
```
Comment (elland) [inline libs/types-common/src/Data/Id.hs:190]: Regex is hard <3

**Review by elland — CHANGES_REQUESTED** [review]
Some docs got mangled b/c of regex replaces.

File: libs/types-common/src/Data/Id.hs (hunk 2 above)
Comment (akshaymankar) [inline libs/types-common/src/Data/Id.hs:190]: collateral regex damange :sweat_smile:

**Review by akshaymankar — COMMENTED** [review]
Looks good apart from the comments being attacked by the regex

File: libs/types-common/src/Data/Id.hs (hunk 2 above)
Comment (battermann) [reply libs/types-common/src/Data/Id.hs:190]: oops

**Comment by elland** [comment]
Love seeing this kind of PR <3

**Pull request merged**